src/elements/content-sidebar/activity-feed/comment/__tests__/Replies.test.js

```diff
@@ -170,6 +172,27 @@ describe('elements/content-sidebar/ActivityFeed/comment/Replies', () => {
         expect(replyCreate).toBeCalledWith('Batman');
     });
 
+    test('should call onReplyDelete when reply is deleted', () => {
+        getWrapper({
+            replies: [comment, comment2],
+        });
+
+        const [menu] = screen.getAllByTestId('comment-actions-menu');
+
+        fireEvent.click(menu);
+
+        expect(screen.getByTestId('delete-comment')).toBeInTheDocument();
```

Comment:
I wonder if we should add and select by a label here. I think we want/need that for a11y.

If not, should we use `getByText`? If we're worried about "Delete" showing up in the comments themselves, we could use `within`.